pmaweb/templates/contractor.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+
+<h2>Hiring a contract developer</h2>
+
+<p>
+The phpMyAdmin project is often looking for developers who are interested in becoming contractors
+performing work on the project. Applicants who do not fit our needs at the time of application
+will be kept on file for a period of one year in the event the requirements change or a new position
+is available. Within a few days of applying, applicants can expect a response confirming that their
+submission was received.
+</p>
+
+<h3>Overview</h3>
+
+<p>
+The phpMyAdmin Project is looking for a full-time or part-time developer to assist in development, including bug fixing and refactoring.
+</p>
+
+<h3>Work plan</h3>
+
+<p>
+The ideal candidate will dynamically balance their workload based on outstanding issues and priority, but is
+anticipated to break down to these percentages (in priority order of what's most important to least important):
+</p>
+
+<ul>
+  <li>Security maintenance (5%)</li>
+  <li>Bug fixing and issue assessment (45%)</li>
+  <li>Code base improvement like refactoring and writing unit tests (45%)</li>
+  <li>Implementation of new features (5%)</li>
```

Comment:
I'm not really sure we should put the percents here - I think it's more reasonable to adjust workload based on current situation. It never really worked for me even though we were adjusting it for my second contract.

What I think is important the priority of things, what is already covered.

Another thing comes to my mind - do we require all contractors to work on security issues?